src/runtime/audience-action-flow.ts

```diff
@@ -321,6 +330,69 @@ export class AudienceActionFlow {
   }
 
   /**
+   * Populates localStorage with PPS configuration parameters based on
+   * SurveyDataTransferRequest.
+   * @param {SurveyDataTransferRequest} request
+   * @return {boolean}
+   * @private
+   **/
+  async configureAnswerPpsData(request: SurveyDataTransferRequest) {
+    const iabAudienceKey = 'iabtaxonomiesvalues';
+    // PPS value field is optional and category may not be populated
+    // in accordance to IAB taxonomies.
+    const ppsConfigParams = request
+      .getSurveyQuestionsList()!
+      .filter(
+        (question) => question.getSurveyAnswersList()![0].getPpsValue() !== null
+      )
+      .map((question) => question.getSurveyAnswersList()![0].getPpsValue());
+
+    if (ppsConfigParams.length === 0) {
+      // Answers with no PPS parameters should always evaluate to true with
+      // the feature flag enabled.
+      return true;
+    }
+
+    const existingIabTaxonomy = await this.storage_.get(
+      iabAudienceKey,
+      /* useLocalStorage= */ true
+    );
+    let existingIabTaxonomyMap;
+    if (!existingIabTaxonomy) {
+      existingIabTaxonomyMap = {
+        '[googletag.enums.Taxonomy.IAB_AUDIENCE_1_1]': ppsConfigParams,
+      };
+    } else {
+      const existingIabMap = this.filterDigits_(existingIabTaxonomy);
+      existingIabTaxonomyMap = {
+        '[googletag.enums.Taxonomy.IAB_AUDIENCE_1_1]': existingIabMap.concat(
+          ppsConfigParams as string[]
+        ),
+      };
+    }
+    await Promise.resolve(
+      this.storage_.set(
+        iabAudienceKey,
+        JSON.stringify(existingIabTaxonomyMap),
+        /* useLocalStorage= */ true
+      )
+    );
+    return await !!Promise.resolve(
+      this.storage_.get(iabAudienceKey, /* useLocalStorage= */ true)
+    );
+  }
+
+  /**
+   * Filters array to contain only an array of digits after the identifier key
+   * in localStorage.
+   * @param {Array} list
+   * @returns
+   */
```

Comment:
As above, the type information should be specified using TS syntax and can be removed from the JSDoc. Looks like this method is also meant to be private.

Reply:
sorry about that! migration happened mid-PR but i'll fix all the js->ts instances